amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/field/APIField.java

```diff
@@ -102,6 +107,12 @@
 
     @JsonProperty(ActivityEPConstants.COMMON_POSSIBLE_VALUES)
     private String commonPossibleValuesPath;
+    
+    @JsonIgnore
+    private TranslationSettings.TranslationType translationType;
+
+    @JsonIgnore
+    private boolean isCollection;
```

Comment:
This is probably needed for cases like ppc_amount where the field is an object in json but object model it is a discriminated collection with just one element. I think this creates unnecessarily complex code in ObjectImporter/Exporter. What if DiscriminatedFieldAccessor returns just a single object for ppc_amount instead of a list with a single element? In that case this field will not be required at all.

Reply:
After investigating the code, it would be difficult at this moment to change the type of DiscriminatedFieldAccessor  without full refactoring of Object Importer. It will be done in a separate ticket.